### PR TITLE
[FEATURE] Flash window on respawn

### PR DESCRIPTION
**Pull request by electricbrass**
Flashes the taskbar any time the player is spawned in-game and the window is not focused.

Address #637

### CODE REVIEW

File: client/sdl/i_video_sdl20.h [hunk 1]
```diff
@@ -171,6 +171,10 @@ class ISDL20Window : public IWindow
 
 	virtual bool isFocused() const;
 
+	#if defined(SDL2016)
+	virtual void flashWindow() const;
```
Comment (leximayfield) [inline client/sdl/i_video_sdl20.h:174]: I don't really like the idea of changing the shape of the class based on the SDL version.  I think a better approach would be for this method to always exist, but ifdef out the actual implementation in the method definition.

File: client/sdl/i_video_sdl20.cpp [hunk 2]
```diff
@@ -712,6 +712,19 @@ bool ISDL20Window::isFocused() const
...
+#if defined(SDL2016)
+
+//
+// ISDL20Window:flashWindow
+//
+// Flashes the taskbar icon if the window is not focused
+//
+void ISDL20Window::flashWindow() const {
+	if (!this->isFocused())
+		SDL_FlashWindow(mSDLWindow, SDL_FLASH_UNTIL_FOCUSED);
```
Comment (leximayfield) [inline client/sdl/i_video_sdl20.cpp:723]: I don't really like the idea of changing the shape of the class based on the SDL version. I think a better approach would be to move the `#ifdef` pair to be inside of the `ISDL20Window::flashWindow()` implementation.

**Review by leximayfield — CHANGES_REQUESTED** [review]
I like the change, but I think it would be cleaner to only require a single ifdef at inside the method definition itself.

**Comment by electricbrass** [comment]
I tried just one but did it wrong and so added the 2nd. I agree this is better.

**Review by leximayfield — APPROVED** [review]
Nice!  LGTM.

**Pull request merged**